components/eamxx/src/share/util/eamxx_repro_sum_mod.F90

```diff
@@ -0,0 +1,31 @@
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+module eamxx_repro_sum_mod
+
+  implicit none
+
+contains
+
+  subroutine eamxx_repro_sum(send, recv, nlocal, nfld, comm) bind(c)
+    use iso_c_binding, only: c_int, c_double
+
+!standalone scream -- does it have access to shared libs?
```

Comment:
Yes. But I don't think it builds `shr_reprosum_mod` in it. You should add it to [this](https://github.com/E3SM-Project/E3SM/blob/620edabb54fda2c88f0d1dbd2ee14e25ab3edaf3/components/eamxx/cmake/tpls/CsmShare.cmake#L28) list.